src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/FeaturedTableServiceTests.cs

```diff
@@ -0,0 +1,1044 @@
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var featuredTableService = SetupFeaturedTableService(
+                context);
+
+            var result = await featuredTableService.Reorder(
+                Guid.NewGuid(),
+                new List<Guid>
+                {
+                    Guid.NewGuid(), Guid.NewGuid(),
+                });
+
+            result.AssertNotFound();
+        }
+    }
+
+    [Fact]
+    public async Task Reorder_ProvidedIdsDifferFromReleaseFeaturedTableIds()
+    {
+        var release = new Release();
+        var dataBlock = new DataBlock();
+        var featuredTable1 = new FeaturedTable
+        {
+            Name = "Featured table name 1",
+            Order = 3,
+            DataBlock = dataBlock,
+            Release = release,
+        };
+        var featuredTable2 = new FeaturedTable
+        {
+            Name = "Featured table name 2",
+            Order = 1,
+            DataBlock = dataBlock,
+            Release = release,
+        };
+        var featuredTable3 = new FeaturedTable
+        {
+            Name = "Featured table name 3",
+            Order = 2,
+            DataBlock = new DataBlock(),
+            Release = release,
+        };
+        var unassociatedFeaturedTable = new FeaturedTable
+        {
+            Name = "Unassociated featured table",
+            Order = 4,
+            DataBlock = new DataBlock(),
+            Release = new Release(),
+        };
+
+        var contextId = Guid.NewGuid().ToString();
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            await context.Releases.AddRangeAsync(release);
+            await context.FeaturedTables.AddRangeAsync(
+                featuredTable1, featuredTable2, featuredTable3, unassociatedFeaturedTable);
+            await context.SaveChangesAsync();
+        }
+
+        await using (var context = InMemoryContentDbContext(contextId))
+        {
+            var dataBlockService = new Mock<IDataBlockService>(MockBehavior.Strict);
+            dataBlockService.Setup(s => s.ListDataBlocks(release.Id))
+                .ReturnsAsync(new List<DataBlock>
+                {
+                    featuredTable1.DataBlock, featuredTable2.DataBlock, featuredTable3.DataBlock
+                });
+
+            var featuredTableService = SetupFeaturedTableService(
+                context,
+                dataBlockService: dataBlockService.Object);
+
+            var result = await featuredTableService.Reorder(
+                release.Id,
+                new List<Guid>
+                {
+                    featuredTable1.Id, featuredTable2.Id, unassociatedFeaturedTable.Id,
+                });
+
+            result.AssertBadRequest(
+                ProvidedFeaturedTableIdsDifferFromReleaseFeaturedTableIds);
+        }
+    }
+
+    private FeaturedTableService SetupFeaturedTableService(
```

Comment:
Similar to earlier - would just call this `SetupService` to avoid having to rename this if we ever rename the service.